core/src/main/java/com/amplifyframework/predictions/models/TargetText.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+import androidx.annotation.NonNull;
+
+/**
+ * Predictions category's text interpretation returns
+ * inferences regarding specific portions of input text.
```

Comment:
yea naming is hard :/ I do not find either suggestion to be particularly preferable, so I'll leave it as is to avoid renaming frequently. We can probably discuss with some other people to come up with a better name for it as a whole.